duckduckgo/macos-browser
# External beta build

Task/Issue URL: https://app.asana.com/0/inbox/1199237043628108/1200639119745581/1200816229119677/f
Tech Design URL:  https://app.asana.com/0/inbox/1199237043628108/1200815845589652/1200816229119677
CC: @samsymons @tomasstrba 

**Description**:
Adds External Beta Build Scheme with:
- disabled Automatic Updater
- "beta=1" Pixel Parameter
- Changed App Name and Icon
- Usage period limited to 1 month asking to delete the App after the expiration
- App Feedback set to Surveymonkey

**Steps to test this PR**:
1. Validate normal App compiles and works without breakages
2. Validate Automatic Updates are disabled and keys are missing in Info.plist in Beta build
3. Validate External Beta App works without breakages
4. Validate app expiration Alert is shown when System Date set to 1 month in future
5. Validate App Feedback form points to Surveymonkey

**Testing checklist**:


---
###### Internal references:
[Software Engineering Expectations](https://app.asana.com/0/59792373528535/199064865822552)
[Technical Design Template](https://app.asana.com/0/59792373528535/184709971311943)
**When ready for review, remember to post the PR in MM**
